.yarnrc

```diff
@@ -1,2 +1,4 @@
---install.audit true
---add.audit true
+# Disabling temporarily
+# https://github.com/yarnpkg/yarn/issues/7025
+# --install.audit true
+# --add.audit true
```

Comment:
The 503 error is getting more persistent and pretty annoying. Sounds like the payload size of our growing repo could be the reason why.